skbio/io/tests/test_fasta.py

```diff
@@ -154,8 +178,19 @@ def setUp(self):
                  description='detailed description \t\twith  new  lines',
                  quality=[42, 42, 442, 442, 42, 42, 42, 42, 42, 43])],
             {},
-            list(map(get_data_path, ['fasta_multi_seq', 'fasta_max_width_5'])),
-            list(map(get_data_path, ['qual_multi_seq', 'qual_max_width_5']))
+            list(map(get_data_path, ['fasta_multi_seq', 'fasta_max_width_5',
+                                     'fasta_blank_lines_between_records',
+                                     'fasta_ws_lines_between_records'])),
```

Comment:
Nevermind, I see that the fasta files are different than the expected results here. Having the qual files exercising these cases should be sufficient since fasta and qual readers use the same underlying parsing logic.